lib/synapse/service_watcher/zookeeper.rb

```diff
@@ -30,12 +30,17 @@ def initialize(opts={}, synapse)
     end
 
     def start
-      @zk_hosts = @discovery['hosts'].sort.join(',')
+      zk_host_list = @discovery['hosts'].sort
+      # extract cluster name by filtering name of first host
+      # remove domain extents and trailing numbers
+      zk_first_host = zk_host_list.first.split('.').first
+      @zk_cluster = zk_first_host[0..(zk_first_host.rindex(/[^0-9]/) || zk_first_host.length)]
+      @zk_hosts = zk_host_list.join(',')
 
       @watcher = nil
       @zk = nil
 
-      log.info "synapse: starting ZK watcher #{@name} @ hosts: #{@zk_hosts}, path: #{@discovery['path']}"
+      log.info "synapse: starting ZK watcher #{@name} @ cluster #{@zk_cluster} hosts: #{@zk_hosts}, path: #{@discovery['path']}"
```

Comment:
The `zk_hosts` string can get pretty long, and the line might get drop depending on the logging facilities (hint: rsyslog).
I think the `zk_cluster` name is a nice addition, and would be sufficient here.

Reply:
addressed